app/views/posts/show.html.erb

```diff
@@ -1,12 +1,16 @@
 <div class="post_show">
     <div class="post">
-        <p><%= @post.title %></p>
-        <p><%= @post.comment_counter %></p>
+        <h3><%= @post.title %></h3>
+        <p><%= @post.comment_counter %> comments</p>
     </div>
     <p><%= @post.text %></p>
     <div class="comments">
-        <% @post.comments.each do |comment| %>
-            <p><%= comment.text %> By: <%= comment.user.name %></p>
+        <% if @post.comments.empty? %>
+          <p>0 Comment</p>
+        <% else %>
+          <% @post.comments.each do |comment| %>
+            <p><%= comment.text %> By: <%= comment.author_id %></p>
+          <% end %>
```

Comment:
- [ ] On the post, page ı should able to likes number too. Kindly add it. [Wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_single_post.png)

![image](https://user-images.githubusercontent.com/38635158/185517999-2d5535f5-3eef-4b4e-b00f-a927e0219520.png)